stack/__main__.py

```diff
@@ -787,6 +788,17 @@ def find_service_account(kind: str, access_level: str) -> Optional[str]:
             member=f'serviceAccount:{ANALYSIS_RUNNER_SERVICE_ACCOUNT}',
         )
 
+    for access_level, service_account in service_accounts['hail']:
```

Comment:
Would Dataproc or Cromwell service accounts need access,too?

Reply:
Maybe, it's not considered good practice to update external state from within a WDL workflow, though it is convenient. More plausible from Hail. Should I add both as "just in case".